doc/build_apps/install_bin.rst

```diff
@@ -31,25 +31,27 @@ Install
 
 CCF releases are available on the `GitHub repository release page <https://github.com/microsoft/CCF/releases>`_.
 
-The CCF Debian package (``ccf_<version>_amd64.deb``) contains the libraries and utilities to start a CCF service and build CCF applications. CCF can be installed as follows:
+The CCF Debian package (``ccf_<platform>_<version>_amd64.deb``) contains the libraries and utilities to start a CCF service and build CCF applications. CCF can be installed as follows:
 
 .. code-block:: bash
 
-    # Set CCF_VERSION to most recent release
+    # Set CCF_VERSION to most recent LTS release
     $ export CCF_VERSION=$(curl -ILs -o /dev/null -w %{url_effective} https://github.com/microsoft/CCF/releases/latest | sed 's/^.*ccf-//')
+    # Set CCF_PLATFORM to virtual, to allow running on any amd64-compatible hardware
+    $ export CCF_PLATFORM=virtual
```

Comment:
If virtual is the default platform we want to show here, I suggest we write a note to say it's not as secure as the SGX and SNP equivalents?

Reply:
The following instructions start sandbox without arguments, which will default to virtual (something we ought to change, arguably, since each package contains its own sandbox), hence the choice here. I'll add a note anyway.